src/components/HeaderButton/styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const StyledLink = styled.a`
+  padding: 0 20px 0 20px;
+  cursor: pointer;
+  color: #bdbcbc;
```

Comment:
I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes